UnleashClient/api/features.py

```diff
@@ -82,7 +82,7 @@ def get_feature_toggles(
         if resp.status_code == 304:
             return None, etag
 
-        return resp.json(), etag
+        return resp.text, etag
```

Comment:
Shouldn't the method signature response change to `Tuple[str, str]`?

Reply:
It does, changed now! On that notes, apparently we don't have MyPy setup either